### PR TITLE
[DOC-7925] Remove deprecated roles

### PR DESCRIPTION
**Pull request by mdlinville**
[DOC-7925] Remove deprecated roles

- Also remove info about legacy service accounts

### Previews
[releases/cloud.md](https://deploy-preview-18402--cockroachdb-docs.netlify.app/docs/releases/cloud.html)
[src/current/cockroachcloud/authorization.md](https://deploy-preview-18402--cockroachdb-docs.netlify.app/docs/cockroachcloud/authorization.html)
[src/current/cockroachcloud/billing-management.md](https://deploy-preview-18402--cockroachdb-docs.netlify.app/docs/cockroachcloud/billing-management.html)
[src/current/cockroachcloud/ccloud-faq.md](https://deploy-preview-18402--cockroachdb-docs.netlify.app/docs/cockroachcloud/ccloud-faq.html)
[src/current/cockroachcloud/client-certs-dedicated.md](https://deploy-preview-18402--cockroachdb-docs.netlify.app/docs/cockroachcloud/client-certs-dedicated.html)
[src/current/cockroachcloud/cloud-api.md](https://deploy-preview-18402--cockroachdb-docs.netlify.app/docs/cockroachcloud/cloud-api.html)
[src/current/cockroachcloud/cloud-org-sso.md](https://deploy-preview-18402--cockroachdb-docs.netlify.app/docs/cockroachcloud/cloud-org-sso.html)
[src/current/cockroachcloud/configure-cloud-org-sso.md](https://deploy-preview-18402--cockroachdb-docs.netlify.app/docs/cockroachcloud/configure-cloud-org-sso.html)
[src/current/cockroachcloud/configure-scim-provisioning.md](https://deploy-preview-18402--cockroachdb-docs.netlify.app/docs/cockroachcloud/configure-scim-provisioning.html)
[src/current/cockroachcloud/connect-to-a-serverless-cluster.md](https://deploy-preview-18402--cockroachdb-docs.netlify.app/docs/cockroachcloud/connect-to-a-serverless-cluster.html)
[src/current/cockroachcloud/connect-to-your-cluster.md](https://deploy-preview-18402--cockroachdb-docs.netlify.app/docs/cockroachcloud/connect-to-your-cluster.html)
[src/current/cockroachcloud/create-a-serverless-cluster.md](https://deploy-preview-18402--cockroachdb-docs.netlify.app/docs/cockroachcloud/create-a-serverless-cluster.html)
[src/current/cockroachcloud/create-an-account.md](https://deploy-preview-18402--cockroachdb-docs.netlify.app/docs/cockroachcloud/create-an-account.html)
[src/current/cockroachcloud/create-your-cluster.md](https://deploy-preview-18402--cockroachdb-docs.netlify.app/docs/cockroachcloud/create-your-cluster.html)
[src/current/cockroachcloud/frequently-asked-questions.md](https://deploy-preview-18402--cockroachdb-docs.netlify.app/docs/cockroachcloud/frequently-asked-questions.html)
[src/current/cockroachcloud/managing-access.md](https://deploy-preview-18402--cockroachdb-docs.netlify.app/docs/cockroachcloud/managing-access.html)
[src/current/cockroachcloud/plan-your-cluster-serverless.md](https://deploy-preview-18402--cockroachdb-docs.netlify.app/docs/cockroachcloud/plan-your-cluster-serverless.html)
[src/current/cockroachcloud/production-checklist.md](https://deploy-preview-18402--cockroachdb-docs.netlify.app/docs/cockroachcloud/production-checklist.html)
[src/current/cockroachcloud/upgrade-policy.md](https://deploy-preview-18402--cockroachdb-docs.netlify.app/docs/cockroachcloud/upgrade-policy.html)
[src/current/cockroachcloud/upgrade-to-v20.1.md](https://deploy-preview-18402--cockroachdb-docs.netlify.app/docs/cockroachcloud/upgrade-to-v20.1.html)
[src/current/cockroachcloud/upgrade-to-v20.2.md](https://deploy-preview-18402--cockroachdb-docs.netlify.app/docs/cockroachcloud/upgrade-to-v20.2.html)
[src/current/cockroachcloud/upgrade-to-v21.1.md](https://deploy-preview-18402--cockroachdb-docs.netlify.app/docs/cockroachcloud/upgrade-to-v21.1.html)
[src/current/cockroachcloud/upgrade-to-v21.2.md](https://deploy-preview-18402--cockroachdb-docs.netlify.app/docs/cockroachcloud/upgrade-to-v21.2.html)
[src/current/cockroachcloud/upgrade-to-v22.1.md](https://deploy-preview-18402--cockroachdb-docs.netlify.app/docs/cockroachcloud/upgrade-to-v22.1.html)
[src/current/cockroachcloud/upgrade-to-v22.2.md](https://deploy-preview-18402--cockroachdb-docs.netlify.app/docs/cockroachcloud/upgrade-to-v22.2.html)
[src/current/cockroachcloud/upgrade-to-v23.1.md](https://deploy-preview-18402--cockroachdb-docs.netlify.app/docs/cockroachcloud/upgrade-to-v23.1.html)
[src/current/cockroachcloud/upgrade-to-v23.2.md](https://deploy-preview-18402--cockroachdb-docs.netlify.app/docs/cockroachcloud/upgrade-to-v23.2.html)
[src/current/cockroachcloud/use-managed-service-backups.md](https://deploy-preview-18402--cockroachdb-docs.netlify.app/docs/cockroachcloud/use-managed-service-backups.html)
[src/current/cockroachcloud/deploy-a-python-to-do-app-with-flask-kubernetes-and-cockroachcloud.md](https://deploy-preview-18402--cockroachdb-docs.netlify.app/docs/cockroachcloud/deploy-a-python-to-do-app-with-flask-kubernetes-and-cockroachcloud.html)

### CODE REVIEW

File: src/current/_includes/cockroachcloud/cockroachcloud-ask-admin.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 {{site.data.alerts.callout_info}}
-Only [organization administrators]({% link cockroachcloud/authorization.md %}#org-administrator-legacy) and [Cluster Administrators]({% link cockroachcloud/authorization.md %}#cluster-administrator) can create SQL users and issue credentials.
+Only [organization administrators]({% link cockroachcloud/authorization.md %}#org-administrator) and [Cluster Administrators]({% link cockroachcloud/authorization.md %}#cluster-administrator) can create SQL users and issue credentials.
```
Comment (mdlinville) [inline src/current/_includes/cockroachcloud/cockroachcloud-ask-admin.md:2]:
```suggestion
Only [Org Administrators]({% link cockroachcloud/authorization.md %}#org-administrator) and [Cluster Administrators]({% link cockroachcloud/authorization.md %}#cluster-administrator) can create SQL users and issue credentials.
```

File: src/current/cockroachcloud/ccloud-faq.md [hunk 2]
```diff
@@ -17,23 +17,20 @@ Cluster Developer is the minimum access role that can be assigned to a cluster u
...
+Yes, it is possible, and often necessary, to assign more than one role to a user. The default minimum access role Org Member is always assigned to every user as long as they’re a part of the CockroachDB {{ site.data.products.cloud }} organization. Beyond that, every other assigned role is additive to the overall entitlements of a user. Best example of this is the initial user who is by default assigned the Org Member, Org Administrator, and Cluster Admin roles at organization scope when they create the CockroachDB {{ site.data.products.cloud }} organization.
 
 ### Can we follow the least privilege principle by using the roles available in the CockroachDB {{ site.data.products.cloud }} authorization model?
 
-Yes, the roles available in the CockroachDB {{ site.data.products.cloud }} authorization model allow admins to grant only those entitlements to users that are supposed to map to their intended workflows. 
+Yes, the roles available in the CockroachDB {{ site.data.products.cloud }} authorization model allow admins to grant only those entitlements to users that are supposed to map to their intended workflows.
```
Comment (mdlinville) [inline src/current/cockroachcloud/ccloud-faq.md:26]:
```suggestion
Yes, the roles available in the CockroachDB {{ site.data.products.cloud }} authorization model allow Org Administrators to grant only those entitlements to users that are required for their intended workflows.
```

File: src/current/cockroachcloud/ccloud-faq.md [hunk 3]
```diff
@@ -17,23 +17,20 @@ Cluster Developer is the minimum access role that can be assigned to a cluster u
...
 
 Cluster level roles like Cluster Admin, Cluster Operator and Cluster Developer allow users to perform pertinent actions for one or more clusters, while providing differentiation between admin and non-admin entitlements.
 
 ### Is the same authorization model used for both service accounts and human users in a CockroachDB {{ site.data.products.cloud }} organization?
 
-Yes, service accounts and human users utilize the same authorization model. Older service accounts that were created before the new authorization model came into effect may still be assigned the legacy permissions, which can be updated to roles from the new authorization model.
+{% comment %}TODO: Did these get migrated?{% endcomment %}
```
Comment (mdlinville) [inline src/current/cockroachcloud/ccloud-faq.md:32]:
```suggestion
```

File: src/current/cockroachcloud/client-certs-dedicated.md [hunk 4]
```diff
@@ -324,7 +324,7 @@ After this operation is performed, clients can no longer authenticate with certi
 {{site.data.alerts.end}}
 
 {{site.data.alerts.callout_success}}
-Managing the certificate authority (CA) certificate for a CockroachDB {{ site.data.products.dedicated }} cluster requires the [Cluster Administrator]({% link cockroachcloud/authorization.md %}#cluster-administrator) or [Org Administrator (legacy)]({% link cockroachcloud/authorization.md %}#org-administrator-legacy) Organization role.
+Managing the certificate authority (CA) certificate for a CockroachDB {{ site.data.products.dedicated }} cluster requires the [Cluster Administrator]({% link cockroachcloud/authorization.md %}#cluster-administrator)  role.
```
Comment (mdlinville) [inline src/current/cockroachcloud/client-certs-dedicated.md:327]:
```suggestion
The [Cluster Administrator]({% link cockroachcloud/authorization.md %}#cluster-administrator) or [Org Administrator]({% link cockroachcloud/authorization.md %}#org-administrator) Organization role is required to manage the CA certificate for a CockroachDB {{ site.data.products.dedicated }} cluster.
```

File: src/current/cockroachcloud/cloud-api.md [hunk 5]
```diff
@@ -80,8 +80,9 @@ Where `{secret_key}` is the [secret key string you stored when you created the A
 
 To create a cluster, send a `POST` request to the `/v1/clusters` endpoint.
 
+{% comment %}TODO: Did these get migrated? Several other mentions below -- maybe reuse this note{% endcomment %}
```
Comment (mdlinville) [inline src/current/cockroachcloud/cloud-api.md:83]:
```suggestion
```

**Review by pawalt — APPROVED** [review]
Thanks for this!

File: src/current/cockroachcloud/ccloud-faq.md [hunk 6]
```diff
@@ -17,23 +17,19 @@ Cluster Developer is the minimum access role that can be assigned to a cluster u
...
 
-Yes, it is possible, and often necessary, to assign more than one role to a user. The default minimum access role Org Member is always assigned to every user as long as they’re a part of the CockroachDB {{ site.data.products.cloud }} organization. Beyond that, every other assigned role is additive to the overall entitlements of a user. Best example of this is the initial user who is by default assigned the Org Member, Org Administrator, and Cluster Admin roles at organization scope when they create the CockroachDB {{ site.data.products.cloud }} organization. 
+Yes, it is possible, and often necessary, to assign more than one role to a user. The default minimum access role Org Member is always assigned to every user as long as they’re a part of the CockroachDB {{ site.data.products.cloud }} organization. Beyond that, every other assigned role is additive to the overall entitlements of a user. Best example of this is the initial user who is by default assigned the Org Member, Org Administrator, and Cluster Admin roles at organization scope when they create the CockroachDB {{ site.data.products.cloud }} organization.
```
Comment (mdlinville) [inline src/current/cockroachcloud/ccloud-faq.md:22]:
```suggestion
Yes, it is possible, and often necessary, to assign more than one role to a user. The default minimum access role Org Member is always assigned to every user as long as they’re a part of the CockroachDB {{ site.data.products.cloud }} organization. Beyond that, every other assigned role is additive to the overall entitlements of a user. For example, the initial user automatically assigned the Org Member, Org Administrator, Cluster Administrator, and Billing Coordinator roles at organization scope when they create the CockroachDB {{ site.data.products.cloud }} organization.
```

[the remark this answers is not in the document]
Comment (mdlinville) [reply src/current/cockroachcloud/ccloud-faq.md:22]: Good idea, I improved this even a bit more.

**Review by pawalt — APPROVED** [review]

File: src/current/_includes/releases/cloud/2020-05-04.md [hunk 7]
```diff
@@ -3,6 +3,5 @@
 <h3>General changes</h3>
 
 - Updated the layout of the <a href="https://cockroachlabs.cloud/signup?referralId=docs_cc_release_notes" rel="noopener" target="_blank">Sign up</a> page.
-- [CockroachDB {{ site.data.products.cloud }} org admins](https://www.cockroachlabs.com/docs/cockroachcloud/authorization#org-administrator-legacy) can now update their [CockroachCloud Organization](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/architecture/glossary#organization) name.
-- [CockroachDB {{ site.data.products.cloud }} org admins](https://www.cockroachlabs.com/docs/cockroachcloud/authorization#org-administrator-legacy) can now delete their [CockroachCloud Organization](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/architecture/glossary#organization).
-
+- [CockroachDB {{ site.data.products.cloud }} org admins](https://www.cockroachlabs.com/docs/cockroachcloud/authorization#org-administrator) can now update their [CockroachCloud Organization](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/architecture/glossary#organization) name.
```
Comment (mdlinville) [inline src/current/_includes/releases/cloud/2020-05-04.md:6]:
```suggestion
- [CockroachDB {{ site.data.products.cloud }} Org Administrators](https://www.cockroachlabs.com/docs/cockroachcloud/authorization#org-administrator) can now [update their {{ site.data.products.cloud }} organization's name](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/architecture/glossary#organization).
```

File: src/current/_includes/releases/cloud/2020-05-04.md [hunk 8]
```diff
@@ -3,6 +3,5 @@
...
+- [CockroachDB {{ site.data.products.cloud }} org admins](https://www.cockroachlabs.com/docs/cockroachcloud/authorization#org-administrator) can now delete their [CockroachCloud Organization](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/architecture/glossary#organization).
```
Comment (mdlinville) [inline src/current/_includes/releases/cloud/2020-05-04.md:7]:
```suggestion
- [CockroachDB {{ site.data.products.cloud }} Org Administrators](https://www.cockroachlabs.com/docs/cockroachcloud/authorization#org-administrator) can now [delete their {{ site.data.products.cloud }} Organization](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/architecture/glossary#organization).
```

File: src/current/_includes/releases/cloud/2023-03-06.md [hunk 9]
```diff
@@ -1,5 +1,5 @@
 ## March 06, 2023
-  
+
 <h3> Security updates </h3>
 
-- The [**Migrations** page](https://www.cockroachlabs.com/docs/cockroachcloud/migrations-page) is now limited to [admin](https://www.cockroachlabs.com/docs/cockroachcloud/authorization#org-administrator-legacy) users.
+- The [**Migrations** page](https://www.cockroachlabs.com/docs/cockroachcloud/migrations-page) is now limited to [Org Administrators](https://www.cockroachlabs.com/docs/cockroachcloud/authorization#org-administrator) .
```
Comment (kathancox) [inline src/current/_includes/releases/cloud/2023-03-06.md:5]:
```suggestion
- The [**Migrations** page](https://www.cockroachlabs.com/docs/cockroachcloud/migrations-page) is now limited to [Org Administrators](https://www.cockroachlabs.com/docs/cockroachcloud/authorization#org-administrator).
```

File: src/current/_includes/releases/cloud/2024-03-20.md [hunk 10]
```diff
@@ -0,0 +1,7 @@
+## March 20, 2024
+
+<h3 id="2024-03-20-security-updates"> Security updates </h3>
+
+- The following deprecated legacy roles have been removed, and all CockroachDB {{ site.data.products.cloud }} organizations have been migrated to use [fine-grained roles](https://www.cockroachlabs.com/docs/cockroachcloud/authorization#organization-user-roles).
```
Comment (kathancox) [inline src/current/_includes/releases/cloud/2024-03-20.md:5]:
```suggestion
- All CockroachDB {{ site.data.products.cloud }} organizations have been migrated to use [fine-grained roles](https://www.cockroachlabs.com/docs/cockroachcloud/authorization#organization-user-roles). The following deprecated legacy roles have been removed:
```

File: src/current/cockroachcloud/ccloud-faq.md [hunk 11]
```diff
@@ -17,23 +17,19 @@ Cluster Developer is the minimum access role that can be assigned to a cluster u
...
 
-Yes, it is possible, and often necessary, to assign more than one role to a user. The default minimum access role Org Member is always assigned to every user as long as they’re a part of the CockroachDB {{ site.data.products.cloud }} organization. Beyond that, every other assigned role is additive to the overall entitlements of a user. Best example of this is the initial user who is by default assigned the Org Member, Org Administrator, and Cluster Admin roles at organization scope when they create the CockroachDB {{ site.data.products.cloud }} organization. 
+Yes, it is possible, and often necessary, to assign more than one role to a user. The default minimum access role Org Member is always assigned to every user as long as they’re a part of the CockroachDB {{ site.data.products.cloud }} organization. Beyond that, every other assigned role is additive to the overall entitlements of a user. For example, the initial user automatically assigned the Org Member, Org Administrator, Cluster Administrator, and Billing Coordinator roles at organization scope when they create the CockroachDB {{ site.data.products.cloud }} organization.
```
Comment (kathancox) [inline src/current/cockroachcloud/ccloud-faq.md:22]:
```suggestion
Yes, it is possible, and often necessary, to assign more than one role to a user. The default minimum access role Org Member is always assigned to every user as long as they’re a part of the CockroachDB {{ site.data.products.cloud }} organization. Beyond that, every other assigned role is additive to the overall entitlements of a user. For example, the initial user is automatically assigned the Org Member, Org Administrator, Cluster Administrator, and Billing Coordinator roles at organization scope when they create the CockroachDB {{ site.data.products.cloud }} organization.
```

File: src/current/cockroachcloud/ccloud-faq.md [hunk 12]
```diff
@@ -74,4 +70,4 @@ Refer to:
 
 ### How can we track and audit role-assignment actions in a CockroachDB {{ site.data.products.cloud }} organization?
 
-Any user with the Org Administrator role can access [Cloud Organization audit logs]({% link cockroachcloud/cloud-org-audit-logs.md %}) capability to track when users are added and removed in the CockroachDB {{ site.data.products.cloud }} organization, and whenever any role assignment changes are performed for those users.
\ No newline at end of file
+Any user with the Org Administrator role can access [Cloud Organization audit logs]({% link cockroachcloud/cloud-org-audit-logs.md %}) capability to track when users are added and removed in the CockroachDB {{ site.data.products.cloud }} organization, and whenever any role assignment changes are performed for those users.
```
Comment (kathancox) [inline src/current/cockroachcloud/ccloud-faq.md:73]: This sounded weird to me, but I don't know if what I have added fits with the sense either.
```suggestion
Any user with the Org Administrator role can access the [Cloud Organization audit logs]({% link cockroachcloud/cloud-org-audit-logs.md %}) capability to track when users are added and removed in the CockroachDB {{ site.data.products.cloud }} organization, and whenever any role assignment changes are performed for those users.
```

**Review by kathancox — APPROVED** [review]
A couple of _super_ minor nits, but otherwise LGTM!

File: src/current/cockroachcloud/ccloud-faq.md (hunk 12 above)
Comment (mdlinville) [reply src/current/cockroachcloud/ccloud-faq.md:73]: Good idea, I tweaked your suggestion a bit to remove the "capability" indirection entirely.

**Pull request merged**